apps/src/lib/levelbuilder/lesson-editor/LessonEditor.jsx

```diff
@@ -255,6 +258,20 @@ class LessonEditor extends Component {
               </p>
             </HelpTip>
           </label>
+          <label>
```

Comment:
We probably don't want levelbuilders to have to know this, right? Maybe this should just be a level property?

Reply:
+1, discussed a bit offline but seems setting this on the level (or the game?) would be better. I'm wondering if we can even go so far as storing the list of lab-lab compatible app types on the backend (like some list of `game&.lab_lab_enabled?`), and then send that list down to the front end, so that level-switching code doesn't need to store its own list of compatible apps.

Reply:
This is now set on the level.  For `music` levels, `uses_lab_container?` will always be `true`.  For `standalone_video` levels, we can mark it `true` using a serialized attribute for the specific levels that appear alongside `music` levels.  A bonus is that the level-switching code no longer stores its own list of compatible apps, and instead just checks whether the user is moving between two levels that use lab container.